src/core/Akka.Persistence.TCK/Query/CurrentEventsByTagSpec.cs

```diff
@@ -188,7 +214,9 @@ public virtual void ReadJournal_query_CurrentEventsByTag_should_see_all_150_even
         [Fact]
         public void ReadJournal_query_CurrentEventsByTag_should_include_timestamp_in_EventEnvelope()
         {
-            var queries = ReadJournal as ICurrentEventsByTagQuery;
+            if (ReadJournal is not ICurrentEventsByTagQuery queries)
+                throw IsTypeException.ForMismatchedType(nameof(ICurrentEventsByTagQuery), ReadJournal?.GetType().Name ?? "null");
```

Comment:
LGTM - shouldn't run these specs for journals that don't support it / should let the implementor know they've made a boo-boo.